misc/swagger-ballerina/modules/swagger-to-ballerina-generator/src/main/java/org/ballerinalang/swagger/model/BalOpenApi.java

```diff
@@ -82,15 +145,32 @@ public OpenApiWrapper buildFromOpenAPI(OpenAPI openAPI) throws MalformedURLExcep
                 port = port == -1 ? 80 : port;
             }
         }
+    }
+
+    /**
+     * Extract security requirements as a set.
+     *
+     * @param openAPI <code>OpenAPI</code> definition object with security definition
+     */
+    private void setSecurityRequirements(OpenAPI openAPI) {
+        this.security = new LinkedHashSet<>();
+        List<SecurityRequirement> requirements = openAPI.getSecurity();
```

Comment:
Doing null check and checking requirements has element in same line and process would be more human readable. If that failed then we can return.

Reply:
@sanjeewa-malalgoda Did you mean to check if requirements list is not empty?

Reply:
Fixed with 06d87e1